pyface/ui/qt4/util/tests/test_event_loop_helper.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019, Enthought Inc.
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+import unittest
+
+from traits.api import HasTraits, provides
+
+from pyface.i_gui import IGUI
+from pyface.ui.qt4.util.event_loop_helper import EventLoopHelper
```

Comment:
Isn't there a toolkit-agnostic way of getting the `EventLoopHelper`? If there is, we can make this test toolkit-agnostic, which it should be IMO

Reply:
I will do this in a follow up PR since we need to add the generic module for it (similar to #926)